### DISCUSSION TITLE
New device Nous thermometer

### DISCUSSION THREAD

**Discussion by Matriciel** [Koenkk/zigbee2mqtt#21314] [Devices]
Hi,
I have received a new version of Nous SZ-T04 thermometer.
The old witch i have is _TZE200_c7emyjom
The new model is _TZE200_qrztc3ev

If i add the new model in file nous.js, it work perfectly.

@Koenkk, can you add the new reference in the nous.js file ...

Thank you

**Comment by Koenkk** [comment]
done!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Reply by Matriciel** [reply]
Thank you ...
